### ISSUE TITLE
AttributeError: 'Namespace' object has no attribute 'addons'

### ISSUE THREAD

**Issue by mgorny** [pkgcore/pkgcheck#268]
```
$ git rev-parse origin
e194129d701088b60e798631bdb46346d1a4946e
$ git rev-parse master
f6088d565d3e3aff0da992aec1ddaca3934b0e14
$ pkgcheck scan --commits
virtual/dist-kernel
  UnstableOnly: for arch: [ arm ], all versions are unstable: [ 5.10.4, 5.10.5 ]
  UnstableOnly: for arches: [ amd64, arm64, ppc64, x86 ], all versions are unstable: [ 5.4.86, 5.4.87, 5.10.4, 5.10.5 ]
  RedundantVersion: version 5.4.86: slot(0) keywords are overshadowed by versions: 5.4.87, 5.10.4, 5.10.5
  RedundantVersion: version 5.4.87: slot(0) keywords are overshadowed by versions: 5.10.4, 5.10.5
  RedundantVersion: version 5.10.4: slot(0) keywords are overshadowed by version: 5.10.5

dev-python/geventhttpclient
  UnstableOnly: for arches: [ arm, ppc64 ], all versions are unstable: [ 1.4.5 ]
  PotentialStable: version 1.4.5: slot(0), stabled arches: [ amd64, arm64, ppc, x86 ], potentials: [ ~arm, ~ppc64 ]

sys-kernel/vanilla-kernel
  UnstableOnly: for arches: [ amd64, arm64, x86 ], all versions are unstable: [ 5.4.80, 5.4.83, 5.4.85, 5.4.86, 5.4.87, 5.10.3, 5.10.4, 5.10.5 ]
  UnstableOnly: for arches: [ arm, ppc64 ], all versions are unstable: [ 5.10.3, 5.10.4, 5.10.5 ]

sys-kernel/gentoo-kernel-bin
  DroppedKeywords: version 5.10.5: arm64

sys-kernel/gentoo-kernel
  UnstableOnly: for arch: [ arm ], all versions are unstable: [ 5.10.3, 5.10.4, 5.10.5 ]
  LaggingStable: version 5.4.80-r1: slot(5.4.80), stabled arches: [ amd64, x86 ], lagging: [ ~arm64, ~ppc64 ]
  PotentialStable: version 5.4.83: slot(5.4.83), stabled arches: [ amd64, arm64 ], potentials: [ ~ppc64, ~x86 ]
Traceback (most recent call last):
  File "/usr/lib/python3.9/site-packages/snakeoil/obj.py", line 103, in popattr
    return obj.__dict__.pop(name)
KeyError: 'addons'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/lib/python-exec/python3.9/pkgcheck", line 4, in <module>
    scripts.run(basename(__file__))
  File "/usr/lib/python3.9/site-packages/pkgcheck/scripts/__init__.py", line 38, in run
    sys.exit(tool())
  File "/usr/lib/python3.9/site-packages/snakeoil/cli/tool.py", line 79, in __call__
    ret = self.main()
  File "/usr/lib/python3.9/site-packages/pkgcheck/cli.py", line 22, in main
    return super().main()
  File "/usr/lib/python3.9/site-packages/snakeoil/cli/tool.py", line 185, in main
    self.handle_exec_exception(e)
  File "/usr/lib/python3.9/site-packages/snakeoil/cli/tool.py", line 170, in main
    exitstatus = func(self.options, self.out, self.err)
  File "/usr/lib/python3.9/site-packages/pkgcheck/scripts/pkgcheck.py", line 535, in _scan
    pipe = Pipeline(options, scan_scope, restrict)
  File "/usr/lib/python3.9/site-packages/pkgcheck/pipeline.py", line 44, in __init__
    self._pipes = self._create_runners()
  File "/usr/lib/python3.9/site-packages/pkgcheck/pipeline.py", line 60, in _create_runners
    enabled_checks = init_checks(self.options.pop('addons'), self.options)
  File "/usr/lib/python3.9/site-packages/snakeoil/cli/arghparse.py", line 528, in pop
    return popattr(self, key)
  File "/usr/lib/python3.9/site-packages/snakeoil/obj.py", line 108, in popattr
    getattr(obj, name)
  File "/usr/lib/python3.9/site-packages/snakeoil/cli/arghparse.py", line 535, in __getattribute__
    val = super().__getattribute__(name)
AttributeError: 'Namespace' object has no attribute 'addons'

**Comment by radhermit** [comment]
Already fixed in a almost a month ago b545006791b481d0b34362839d1643d18aecacfe.

**Comment by mgorny** [comment]
Oh, I didn't notice I wasn't using the live version anymore. Sorry.
